contributingGuides/STYLE.md

```diff
@@ -413,63 +411,7 @@ const propTypes = {
 };
 ```
 
-* Avoid public methods on components and calling them via refs
```

Comment:
This defensive comment is largely irrelevant if we are removing class components. The way to do this with hooks would be [`useImperativeHandle()`](https://react.dev/reference/react/useImperativeHandle#useimperativehandle), but it's kind of a rare edge case that we would reach for it at all. And generally much harder to abuse this without knowing that it's an option.